server/main.lua

```diff
@@ -1434,15 +1457,29 @@ RegisterServerEvent("mdt:server:AddLog", function(text)
 	AddLog(text)
 end)
 
-function GetBoloStatus(plate)
+function GetBoloStatus(plate, title, boloid)
     local result = MySQL.query.await("SELECT * FROM mdt_bolos where plate = @plate", {['@plate'] = plate})
 	if result and result[1] then
-		return true
+		local title = result[1]['title']
+		local boloid = result[1]['id']
+		return true, title, boloid

```

Comment:
Why are you passing in title and boloId arguments, but also reading them from the result of query? You can just use the values from the query result and remove the added args

Reply:
Amended code in new commit d13ea57